x/rollappparams/types/params.go

```diff
@@ -0,0 +1,125 @@
+package types
+
+import (
+	"fmt"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/dymensionxyz/dymint/da/registry"
+)
+
+var (
+	DefaultDA               = "celestia"
+	DefaultCommit           = "74fad6a00713cba62352c2451c6b7ab73571c515"
+	DefaultMaxBlockGas      = 400000000
```

Comment:
everything which is const should be under const

Reply:
done